CHANGES.md

```diff
@@ -1,6 +1,14 @@
 Changes
 =======
 
+v1.1.3
+------
+
+### Improvements
+
+- Bumped `node` version from `node12` to `node16`, in response to
+  deprecation warnings.
+
```

Comment:
[consider][minor] We could add the missing release note for v1.1.2.

Reply:
Hmm, i suppose there isn't a reason that we shouldn't.

I'm a tad dubious because as far as I can tell, the package.json files wern't updated to have the `1.1.2` version string at point of tagging. Not sure how that interacts, if there actually _was_ a 1.1.2 release or not.

Reply:
Went ahead and added the note. It's listed as a bug fix in the release itself, so i've kept that, although i don't think it really is one. [5195a30](https://github.com/TheFoundryVisionmongers/fn-pylint-action/pull/9/commits/5195a303238abffb50f15e5fd07b73b4510fa6f9)

Reply:
I agree it's not really a bug fix, unless the "bug" is us using an older (and so potentially insecure) version of an Action, which is grasping at straws, but fine.